tests/test_sync.py

```diff
@@ -349,8 +350,7 @@ def test_close_open_template_merge_prs(self, mock_get):
             for pr in prs:
                 if pr["state"] != "open":
                     continue
```

Comment:
```suggestion
                if pr["state"] == "open":
                    mock_close_open_pr.assert_any_call(pr)
```
This is probably the better way to write this

Reply:
yup. that's better.